frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineResources.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PipelineForm from '../../pipeline-form/PipelineForm';
+import PipelineResources from '../PipelineResources';
```

Comment:
Same issue here... we are repeating ground with tests and that is the last thing you want to do as it'll make multiple tests flag failed when it's unrelated to them.